recipes/radical.saga/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "radical.saga" %}
+{% set version = "0.60.0" %}
+{% set sha256 = "66045355acb70b287f63d439a929ef61c1404d83049640fbd9e0170e8e36f9f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [py3k or win]
```

This looks like it could be `noarch`?
```suggestion
  noarch: python
```

---

We will in one of the next releases add a part of the module that needs compilation.  Does that change anything wrt. your suggestion?  Otherwise happy to accept that.

---

In addition, we do not support windows with RADICAL-SAGA and currently there is no such a requirement. As a result, it cannot be `noarch`.

---

Lack of Windows support isn't an issue for `noarch` but if you're going to need to have compiled code in the near future it's better to leave it as is.